docker/pytorch-aarch64/scripts/build-torchdata.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # *******************************************************************************
-# Copyright 2021-2022 Arm Limited and affiliates.
```

Comment:
Of course I don't know what advice / policies your employer has on contributions to OSS, but it may be worth confirming. As far as I'm aware, (c) notice would typically include all the contributors, for example - https://github.com/oneapi-src/oneDNN/blob/adb5654c0f9e5bc446a8035a0a749ca495c14add/.github/automation/.drone.yml#L2-L3

Reply:
I'm checking on the header license part.

Reply:
Ok, thanks.